src/tic_tac_toe/computer.clj

```diff
@@ -1,17 +1,49 @@
 (ns tic-tac-toe.computer
-  (:require [tic-tac-toe.player :as player]
+  (:require [tic-tac-toe.board :as board]
+            [tic-tac-toe.player :as player]
+            [tic-tac-toe.referee :as referee]
             [tic-tac-toe.user-interface :as ui]))
 
-(defn- first-available
-  [board]
-  (first (keep-indexed #(if (= 'empty %2) %1) board)))
+(def max-depth 9)
```

Comment:
at this point does the max depth do anything for us? The most moves we can have on a board is 9

Reply:
No.  At this point, it's pretty much creating duplication that `draw?` via `game-over?` can handle instead.  I've removed it in case larger boards are not requested.